Makefile

```diff
@@ -25,7 +25,7 @@ NSM_PATH?=${TOP}/../networkservicemesh
 CLUSTER_RULES_PREFIX?=kind
 PREFIX?=k8s
 CONTAINER_BUILD_PREFIX?=docker
-VPP_AGENT?=ligato/vpp-agent:v2.3.0
+VPP_AGENT?=artembelov/vpp-agent:v2.5.1
```

Comment:
Not sure where this version is from... would be best to use an official ligato version.  Could we try https://github.com/networkservicemesh/examples/blob/master/Makefile#L28

Reply:
I see that there's more changes required to move to ligato v3.